pkg/cmd/inspect/resource.go

```diff
@@ -45,6 +48,17 @@ func InspectResource(info *resource.Info, context *resourceContext, o *InspectOp
 
 	switch info.ResourceMapping().Resource.GroupResource() {
 	case configv1.GroupVersion.WithResource("clusteroperators").GroupResource():
+		// first, gather config.openshift.io resource data
+		errs := []error{}
+		if err := o.gatherConfigResourceData(path.Join(o.baseDir, "/cluster-scoped-resources/config.openshift.io"), context); err != nil {
+			errs = append(errs, err)
+		}
```

Comment:
Will we possibly want this collected outside of clusteroperators? 
What other top-level objects do we have a need to collect config.openshift.io for? 

If we add this in with other major objects, will it be skipped because of `ctx.visited.Has`?

Reply:
We could collect it for namespaces as well, if that makes sense.

> If we add this in with other major objects, will it be skipped because of ctx.visited.Has?

It will always be collected once per *resourceContext.

Reply:
No, with this patch, we are only collecting config.openshift.io and operator.openshift.io when a user specifies a `clusteroperator` among the resource/name pairs they wish to collect via the cli.

If we also wish to gather config.openshift.io and operator.openshift.io data when, say, only a namespace is specified by the user, we can copy this block into the `namespaces` switch case as well